Manager/BackupManager.php

```diff
@@ -77,28 +77,13 @@ public function execute()
         try {
             // If we catch an exception or not, we would still like to try cleaning up after us
             $this->logger->info('[dizda-backup] Cleaning up after us.');
-            $this->processor->cleanUp();
+            $this->processorManager->cleanUp();
         } catch (IOException $e) {
-            $this->logger->error('[dizda-backup] Cleaning up failed.');
+            $this->logger->error('[dizda-backup] Cleaning up failed.', array('exception' => $e));
 
             return false;
         }
 
         return $successful;
     }
-
-    public function getClientManager()
-    {
-        return $this->cm;
-    }
-
-    public function getDatabaseManager()
-    {
-        return $this->dbm;
-    }
-
-    public function getProcessorManager()
```

Comment:
As a note from me, the getters for the CM and DM was only for if future users might find them useful (can only think of a situation were they might want to add a self made DB or Client interface that is not part of the bundle, then again maybe a contribution of the interface to the bundle might be a better idea).